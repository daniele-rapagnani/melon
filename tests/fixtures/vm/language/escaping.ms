io.print("\"\"");
io.print("First line\nSecond line");
io.print("\t\tColumn");
io.print("Hell\u{6F}\u{20}world!");