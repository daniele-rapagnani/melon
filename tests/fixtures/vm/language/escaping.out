""
First line
Second line
		Column
Hello world!
